local-setup/jupyter/dataloader/kubectl_api.py

```
@@ -0,0 +1,245 @@
+    return base64.b64decode(result.stdout).decode()
+
+
+def run_sql(sql: str, env: str = 'chakshu') -> dict:
+    """Run SQL command via kubectl exec"""
+    config = DB_CONFIG.get(env, DB_CONFIG['chakshu'])
+    db_pass = get_db_password(env)
+
+    conn_str = f"postgresql://{config['user']}:{db_pass}@{config['host']}:5432/{config['name']}"
+
+    # Ensure cleanup pod exists
+    subprocess.run(
```

Ruff (0.15.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)